docs/platforms/apple/guides/ios/session-replay/index.mdx

```diff
@@ -103,5 +103,8 @@ Errors that happen on the page while a replay is running will be linked to the r
 Q: Does Session Replay work with SwiftUI?
 A: Yes. It works with both UIKit and SwiftUI.
 
+Q: Why are parts of my replay not masked?
+A: Text views, input views, images, video players and webviews are all masked by default. Images with bundled assets, aren't masked because the likelihood of these assets containing PII is low. If you encounter a view that should be masked by default, consider opening a [GitHub issue](https://github.com/getsentry/sentry-cocoa/issues).
```

Comment:
```suggestion

A: Text views, input views, images, video players and webviews are all masked by default. Images with bundled assets aren't masked because the likelihood of these assets containing PII is low. If you encounter a view that should be masked by default, consider opening a [GitHub issue](https://github.com/getsentry/sentry-cocoa/issues).
```